### PR TITLE
Merge release/lavida (4.5.0) branch into master

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description
**Requires closing  #150 **

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
@MishkaMN

### CODE REVIEW

**Review by MishkaMN — APPROVED** [review]

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,32 +1,3 @@
...
-
-# Install Eigen3
-RUN git clone --depth 1 --branch 3.4.0 https://gitlab.com/libeigen/eigen.git /tmp/eigen \
-    && cd /tmp/eigen \
-    && cmake -B build -DCMAKE_BUILD_TYPE=Release -DEIGEN_BUILD_TESTING=OFF -DEIGEN_BUILD_DOC=OFF \
-    && cmake --build build \
-    && cmake --install build \
-    && cd / \
-    && rm -rf /tmp/eigen
+FROM ghcr.io/usdot-fhwa-stol/carma-builds-x64:focal
```
Comment (MishkaMN) [inline .devcontainer/Dockerfile:1]: is this going to work still? I think now we need carma-system-4.5.0 right?

Comment (MishkaMN) [reply .devcontainer/Dockerfile:1]: I guess this one is .devcontainer so it doesnt matter prolly

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+jobs:
+  build:  # Define a job named 'build'
+    defaults:  # Set default values for the job
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+    runs-on: ubuntu-latest  # Run on the latest version of Ubuntu
+
+    container:  # Use a Docker container as the execution environment
+      image: ghcr.io/usdot-fhwa-stol/carma-builds-x64:focal  # Docker image to use
```
Comment (MishkaMN) [inline .github/workflows/ci.yml:16]: is this going to work still? I think now we need carma-system-4.5.0 right?

**Review by MishkaMN — CHANGES_REQUESTED** [review]

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (MishkaMN) [reply .devcontainer/Dockerfile:1]: from conversation on https://github.com/usdot-fhwa-stol/carma-streets/pull/411
I am still not sure if stays the same or needs to have carma-system tag included

**Review by adamlm — APPROVED** [review]
Looks good pending Misheel's concerns.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (paulbourelly999) [reply .devcontainer/Dockerfile:1]: It is devcontain so it does not matter

File: .github/workflows/ci.yml (hunk 2 above)
Comment (paulbourelly999) [reply .github/workflows/ci.yml:16]: Actually I will make the change if only to guarantee a stable image

**Review by MishkaMN — APPROVED** [review]

**Pull request merged**